x/emissions/keeper/msg_server_withdraw_emissions.go

```diff
@@ -0,0 +1,43 @@
+package keeper
+
+import (
+	"context"
+	"fmt"
+
+	errorsmod "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/zeta-chain/zetacore/cmd/zetacored/config"
+	"github.com/zeta-chain/zetacore/x/emissions/types"
+)
+
+// WithdrawEmission create a withdraw emission object , which is then process at endblock
+// The withdraw emission object is created and stored
+// using the address of the creator as the index key ,therefore, if more that one withdraw requests are created in a block on thr last one would be processed.
+// Creating a withdraw does not guarantee that the emission will be processed
+// All withdraws for a block are deleted at the end of the block irrespective of whether they were processed or not.
+func (k msgServer) WithdrawEmission(goCtx context.Context, msg *types.MsgWithdrawEmission) (*types.MsgWithdrawEmissionResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	// check if the creator address is valid
+	address, err := sdk.AccAddressFromBech32(msg.Creator)
```

Comment:
Already checked in the message validation. 

If we consider this state transition could be called outside of external message, we can replace this check by calling `msg.Validate` instead

Reply:
I do not see any harm in keeping the check, we need to parse the address anyway

Reply:
Ok, fine, didn't realize address is used